crates/ruma-client-api/src/lib.rs

```diff
@@ -13,3 +13,10 @@ pub mod r0;
 pub mod unversioned;
 
 pub use error::Error;
+
+// Wrapper around `Arc<str>` that cannot be used in a meaningful way outside of
```

Comment:
Shouldn't this be `Box<str>`?

Reply:
Hah, yeah. For some reason I had `Arc<str>` first and when I changed it didn't update the comment.